vtex-apps/quickorder
# [Enhancement] Adding an option as a prop to make the columns of the review block hideable

#### What does this PR do? \*

Creates the option of hidding any column in the reviewBlock table through a prop called "hiddenColumns" (an array of strings) in the upload and textArea blocks. If the new prop is not given, the default used is [] keeping the previous behavior of showing all columns.

#### How to test it? \*

Send the prop "hiddenColumns" (an array of strings) with the name of one of the columns as its content (i.e. "seller") and check in the review block if this columns  is not being shown

#### Describe alternatives you've considered, if any. \*

None

#### Related to / Depends on \*

None
